pkg/releaseutil/kind_sorter_test.go

```diff
@@ -329,3 +330,144 @@ func TestKindSorterForHooks(t *testing.T) {
 		})
 	}
 }
+
+func TestKindSorterFailedForCRsUsingAlphabeticalOrder(t *testing.T) {
```

Comment:
Is there a reason that `TestKindSorter` wasn't updated? It already has [one custom king](https://github.com/helm/helm/blob/561cc4280830af4a40f2e60c5f17fb3ee69b4e39/pkg/releaseutil/kind_sorter_test.go#L68-L71) in it. I would like to see and have tests of the sorting in the broader context rather than solely focused on CRs. This is a touchy area.

Reply:
Test cases have been updated, please review, thanks.